developers.diem.com/README.md

```diff
@@ -38,20 +38,14 @@ Read our [Contributing guide](https://developers.diem.com/docs/community/contrib
 * [Diem: The Path Forward](https://developers.diem.com/blog/2019/06/18/the-path-forward/)
 
 ### Diem Codebase
-
-* [Diem Core Overview](https://developers.diem.com/docs/diem-core-overview)
-* [Admission Control](https://developers.diem.com/docs/crates/admission-control)
-* [Bytecode Verifier](https://developers.diem.com/docs/crates/bytecode-verifier)
-* [Consensus](https://developers.diem.com/docs/crates/consensus)
-* [Crypto](https://developers.diem.com/docs/crates/crypto)
-* [Execution](https://developers.diem.com/docs/crates/execution)
-* [Mempool](https://developers.diem.com/docs/crates/mempool)
-* [Move IR Compiler](https://developers.diem.com/docs/crates/ir-to-bytecode)
-* [Move Language](https://developers.diem.com/docs/crates/move-language)
-* [Network](https://developers.diem.com/docs/crates/network)
-* [Storage](https://developers.diem.com/docs/crates/storage)
-* [Virtual Machine](https://developers.diem.com/docs/crates/vm)
-
+* [Diem Core Overview](https://developers.diem.com/docs/core/overview)
```

Comment:
This (line 41) actually does exist. So maybe move this line up to under `Learn about Diem`, which would be on line 23.